### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PisarskyiDev**

### CODE REVIEW

File: taxi/urls.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from taxi.views import index
```
Comment (olena-hul) [inline taxi/urls.py:3]: Fix tests

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: taxi/urls.py (hunk 1 above)
Comment (Soobig666) [reply taxi/urls.py:3]: > 

Done!

**Comment by Soobig666** [comment]
Снимок экрана 2023-03-01 в 13 00 29

File: taxi_service/urls.py [hunk 2]
```diff
@@ -13,9 +13,18 @@
...
 urlpatterns = [
     path("admin/", admin.site.urls),
-]
+    path("", include("taxi.urls", namespace="taxi"))
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+
+
+# urlpatterns = [path("admin/", admin.site.urls),
+#                path("", include("taxi.urls", namespace="taxi"))] + static(
+#                settings.STATIC_URL, document_root=settings.STATIC_ROOT)
```
Comment (olena-hul) [inline taxi_service/urls.py:30]: Remove if not needed

File: templates/base.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    {% block sidebar %}
+      {% include 'sidebar.html' %}
+    {% endblock %}
+  </div>
+  <div>
+    {% block content %}
+    {% endblock %}
+  </div>
+</body>
+</html>
```
Comment (olena-hul) [inline templates/base.html:24]: Add empty line. Fix in all places where needed

**Review by olena-hul — CHANGES_REQUESTED** [review]

File: taxi_service/urls.py (hunk 2 above)
Comment (Soobig666) [reply taxi_service/urls.py]: done!

File: templates/base.html (hunk 3 above)
Comment (Soobig666) [reply templates/base.html]: done!

**Review by olena-hul — APPROVED** [review]

**Comment by olena-hul** [comment]
Good job!

**Pull request closed without merging**